libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/FallbackVersionsObserver.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.navigation.core.trip.session
+
+internal interface FallbackVersionsObserver {
```

Comment:
This is just internal, why not use the `com.mapbox.navigator.FallbackVersionsObserver` internally instead of making a copy?

Reply:
we can set only 1 observer to navigator, so, if we don't use internal copy, we can't observe fallback event from multiple places. Right now we subscribe only `MapboxNavigation`, but we might add observers.

Reply:
yea I agree to keep a local list of observers, but we can use the same interface that Nav Native exposes instead of making a copy of the interface